pygpu/tests/tests_array_tools.py

```diff
@@ -0,0 +1,45 @@
+from pygpu.array_tools import (tril, triu)
+from .support import (gen_gpuarray, context)
+import numpy
+
+
+def test_triu_inplace():
+    ac, ag = gen_gpuarray((10, 10), 'float32', ctx=context)
```

Comment:
As all your tests run with square matrices, there are some cases not tested which will currently raise errors. I suggest you test non-square matrices instead in all your 6 tests. Or a square-matrix AND a non-square matrix in each test.